.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -27,17 +27,32 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
 
+      - name: Setup node
+        uses: actions/setup-node@v3
+
+      - name: Install docs build dependencies
+        run: npm install -g @mermaid-js/mermaid-cli
+
+      - name: Prepare Mermaid assets for docs
+        run: |
+          for file in assets/*.mmd; do
+            mmdc -i "$file" -o "${file/.mmd/.png}"
+          done
+        working-directory: ${{ env.PROJECT_DIR }}
+        shell: bash
+
       - name: Generate docs
-        run: dbt docs generate --target "$TARGET"
+        run: dbt docs generate --target prod
```

Comment:
Based on my experimentation, it turns out docs generation doesn't work on CI branches ([sample run here](https://github.com/ccao-data/data-architecture/actions/runs/6014000527/job/16312738442#step:8:1)). I think there's a bug with the `--defer` behavior in `dbt docs generate`, but it doesn't feel like a super high priority to figure out since GitHub Pages doesn't yet support PR deploy previews so we need to deploy to the prod site anyway.